content/docs/operating/security.md

```diff
@@ -14,13 +14,15 @@ attack vectors that some configurations may enable.
 
 As with any complex systems it is not possible to guarantee that there are no
 bugs. If you find a security bug, please file it in the issue tracker of the
-relevant component.
+relevant component. If you prefer to report security issues in secret, you may
```

Comment:
I'd prefer not spam the team list with misunderstandings of our security model.

Reply:
While I agree with that sentiment, we need to have a defined way for non-public disclosure.

Reply:
We don't need one, we merely need to document how we accept security issues.